src/components/material/MaterialAvailabilityText/online/MaterialAvailabilityTextOnline.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import {
+  useGetV1ProductsIdentifier,
+  useGetV1UserLoansIdentifier
+} from "../../../../core/publizon/publizon";
+import { useText } from "../../../../core/utils/text";
+import MaterialAvailabilityTextParagraph from "../generic/MaterialAvailabilityTextParagraph";
+
+interface MaterialAvailabilityTextOnlineProps {
+  isbn: string;
+}
+
+const MaterialAvailabilityTextOnline: React.FC<
+  MaterialAvailabilityTextOnlineProps
+> = ({ isbn }) => {
+  const t = useText();
+  // const costfreeID = "9788711321683";
+  // const notCostfreeID = "9788740047905";
```

Comment:
Please add a `TODO` comment explaining why is nice to have this code around.

Normally I would argue against having code in comments but in this situation it is necessary since we do not get proper ids in the data.